frontendVue3/src/components/FactLabel.vue

```diff
@@ -0,0 +1,1120 @@
+<template>
+    <v-card>
+        <v-divider class="border-opacity-100"></v-divider>
+        <!--  Data Table for MetaData: -->
+        <v-data-table :headers="factHeaders" :items="metadata" item-value="name">
+            <template v-slot:headers>
+            </template>
+            <template v-slot:top>
+                <v-toolbar flat>
+                    <v-toolbar-title>{{ name }}</v-toolbar-title>
+                    <a :href="fmHref" target="_blank" rel="noopener noreferrer">
```

Comment:
Used the rel stuff because its recommended https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#security_and_privacy